### PR TITLE
feat: add cassandra components

### PR DESCRIPTION
**Pull request by jordanrfrazier**
Adds Cassandra components to the store.

The SVG is from https://www.svgrepo.com/svg/353540/cassandra

### CODE REVIEW

**Comment by jordanrfrazier** [comment]
Preview of icon:
Screenshot 2024-06-03 at 11 32 13 AM

File: src/backend/base/langflow/components/memories/AstraDBMessageReader.py [hunk 1]
```diff
@@ -63,20 +62,12 @@ def get_messages(self, **kwargs) -> list[Record]:
...
         api_endpoint: str,
         namespace: Optional[str] = None,
     ) -> list[Record]:
-        try:
-            pass
-        except ImportError:
-            raise ImportError(
-                "Could not import langchain Astra DB integration package. "
-                "Please install it with `pip install langchain-astradb`."
-            )
```
Comment (jordanrfrazier) [inline src/backend/base/langflow/components/memories/AstraDBMessageReader.py:78]: Import above - this is an artifact of a recent large refactoring

**Comment by ogabrielluiz** [comment]
Wow! 
Thanks @jordanrfrazier 

This is a big one! 

I'll review it ASAP.

File: src/backend/base/langflow/components/memories/AstraDBMessageReader.py (hunk 1 above)
Comment (ogabrielluiz) [reply src/backend/base/langflow/components/memories/AstraDBMessageReader.py:78]: Is this not needed anymore?

Comment (jordanrfrazier) [reply src/backend/base/langflow/components/memories/AstraDBMessageReader.py:78]: Ah, good question. I think you're correct, I should move the import here actually. The `src/backend` poetry file does not directly import `langchain_astradb`, so for users running in backend-only mode, this check can be helpful. I'll re-add this

File: tests/test_database.py [hunk 2]
```diff
@@ -13,7 +11,6 @@
 from langflow.services.database.models.flow import Flow, FlowCreate, FlowUpdate
 from langflow.services.database.utils import session_getter
 from langflow.services.deps import get_db_service
-from langflow.services.settings.base import Settings
```
Comment (jordanrfrazier) [inline tests/test_database.py:16]: Ran `make format` and fixed a few random files. Can revert if you'd like

File: src/backend/base/pyproject.toml [hunk 3]
```diff
@@ -63,6 +63,7 @@ cryptography = "^42.0.5"
 asyncer = "^0.0.5"
 pyperclip = "^1.8.2"
 uncurl = "^0.0.11"
+cassio = "^0.1.7"
```
Comment (ogabrielluiz) [inline src/backend/base/pyproject.toml:66]: I think all you gotta do is move this over to the root of pyproject.toml and it should be good to go. 

Thank you!

**Review by ogabrielluiz — CHANGES_REQUESTED** [review]

File: src/backend/base/pyproject.toml (hunk 3 above)
Comment (jordanrfrazier) [reply src/backend/base/pyproject.toml]: Awesome, thanks @ogabrielluiz. Should be good for another look!

**Comment by jordanrfrazier** [comment]
Hi @ogabrielluiz just want to check that the failing tests are expected (or at least not due to my changes), and that we're not waiting on any further updates? Thanks

**Pull request merged**